app/controllers/application_controller.rb

```diff
@@ -14,6 +14,10 @@ def after_sign_in_path_for(user)
     stored_location_for(user) || helpers.profile_dashboard_url(user)
   end
 
+  def set_success_message(title: "Success", heading: "", content: "")
```

Comment:
Where does this get called then?

Reply:
I imagine in any controller inheriting form application controller?